docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -519,6 +520,54 @@ test('Test2', async t => {
 });
 ```
 
+## Enabling and Disabling Page Reloads
+
+TestCafe tests are independent from each other: each test opens a tested webpage in a browser, simulates test action, and closes the webpage. If you run an entire fixture or a set of tests for the same page, the browser loads this page every time an individual test starts. However, there are cases when you may need to not reload a tested page for each test, but save the page's state between tests. For example, if you test a single-page application or have a group of tests that should be run in a desired order with saving the page's state.
+
+To address these scenarios, TestCafe provides options that allow you to disable or enable page reloads between tests: the [--disable-page-reloads](../using-testcafe/command-line-interface.md#--disable-page-reloads) command line option and the [runner.run](../using-testcafe/programming-interface/runner.md#run) method's `disablePageReloads` option. These options affect all tests to be run.
+
+You can also use the `disablePageReloads` and `enablePageReloads` methods in test code to disable/enable page reloads for fixture's tests or an individual test.
+
+```text
+fixture.disablePageReloads
+```
+
+```text
+fixture.enablePageReloads
+```
+
+```text
+text.disablePageReloads
+```
+
+```text
+text.enablePageReloads
```

Comment:
`test.disablePageReloads`. What reload will be skipped, before this test or after? Should we describe this unobvious moment?